integration/Jenkinsfiles/utest_aurora.groovy

```diff
@@ -0,0 +1,88 @@
+                // can't work in a pipeline multi-branch with build PR because BRANCH_NAME contains PR-XXXXX
+                //[$class: 'StringParameterDefinition', name: 'BRANCH', description: '', defaultValue: BRANCH_NAME],
+                [$class: 'StringParameterDefinition', name: 'PARENT_BRANCH', description: '', defaultValue: 'master'],
+                [$class: 'StringParameterDefinition', name: 'DBPROFILE', description: '', defaultValue: 'pgsql'],
+                [$class: 'StringParameterDefinition', name: 'DBVERSION', description: '', defaultValue: 'unknown'],
+                [$class: 'BooleanParameterDefinition', name: 'CLEAN', description: '', defaultValue: true],
+                [$class: 'StringParameterDefinition', name: 'NODELABEL', description: '', defaultValue: 'REFBENCH']
+        ]]])
+
+currentBuild.setDescription("Branch: $BRANCH -> $PARENT_BRANCH, DB: aurora-$DBPROFILE, VERSION: $DBVERSION")
+
+node(env.NODELABEL) {
+    tool name: 'ant-1.9', type: 'ant'
+    tool name: 'java-8-openjdk', type: 'hudson.model.JDK'
+    tool name: 'maven-3', type: 'hudson.tasks.Maven$MavenInstallation'
+
+    timeout(time: 3, unit: 'HOURS') {
+        timestamps {
+            def sha = stage('clone') {
+                if (CLEAN) {
+                    deleteDir()
+                }
+                checkout(
+                        [$class: 'GitSCM',
+                         branches: [[name: '*/${BRANCH}']],
+                         browser: [$class: 'GithubWeb', repoUrl: 'https://github.com/nuxeo/nuxeo'],
+                         doGenerateSubmoduleConfigurations: false,
+                         extensions: [
+                                 [$class: 'CleanBeforeCheckout'],
+                                 [$class: 'CloneOption', noTags: true, reference: '', shallow: false]
+                         ],
+                         submoduleCfg: [],
+                         userRemoteConfigs: [[url: 'git@github.com:nuxeo/nuxeo.git']]
+                        ])
+                def originsha = sh(returnStdout: true, script: 'git rev-parse HEAD').trim()
+                sh "./clone.py $BRANCH -f $PARENT_BRANCH"
+                return originsha
+            }
+
+            try {
+                stage('tests') {
+                    withBuildStatus("utest/aurora-$DBPROFILE-$DBVERSION", 'https://github.com/nuxeo/nuxeo', sha, "${BUILD_URL}") {
+                        withEnv(["NX_DB_PORT=5432", "NX_DB_ADMINNAME=nuxeoAurora"]){
+                            withCredentials([usernamePassword(credentialsId: 'AURORA_PGSQL', usernameVariable: 'NX_DB_ADMINUSER', passwordVariable: 'NX_DB_ADMINPASS')]) {
+                                try {
+                                    sh "mvn -B -f $WORKSPACE/pom.xml install -Pqa,addons,customdb,$DBPROFILE -Dmaven.test.failure.ignore=true -Dnuxeo.tests.random.mode=STRICT"
```

Comment:
add `-V` to the command line (new recommended usage to ease later debug)

Reply:
No